pages/learn/index.vue

```diff
@@ -16,7 +16,7 @@
       :features="[
         {
           descriptionParagraphs: [
-            'We are building a new home for the Qiskit Textbook on IBM Quantum, and we will be migrating courses and tutorials in the coming months.',
+            'We are building a new home for the Qiskit Textbook on IBM Quantum, and we will be migrating courses and tutorials over the coming months. After December 1, 2023, you can continue to access this page’s content on the Qiskit Github.',
```

Comment:
FYI --- This text says "December 1", but the title "Nov 29". I've asked if that is correct and I'm waiting for an answer

Reply:
November 29th is the correct date!

Reply:
Thank you, Yaiza!

Reply:
https://github.com/Qiskit/qiskit.org/commit/195a3aa48ff66488057430ba2d2987cce634d442